Assets/Mirror/Examples/Room/Scripts/NetworkRoomPlayerExt.cs

```diff
@@ -11,20 +11,6 @@ public override void OnStartClient()
             if (LogFilter.Debug) Debug.LogFormat("OnStartClient {0}", SceneManager.GetActiveScene().name);
 
             base.OnStartClient();
-            NetworkRoomManager room = NetworkManager.singleton as NetworkRoomManager;
```

Comment:
is there a point overriding the method anymore?  can't we not just get rid of the method altogether?

Reply:
Coming in a separate PR, yes, but unrelated to the DDOL change of this PR.